.github/workflows/test.yml

```diff
@@ -0,0 +1,118 @@
+name: Testing workflows for colcon-bundle
+on:
+  pull_request:
+  schedule:
+    - cron: '43 17 * * *'
+
+jobs:
+  tox_unittest:
+    runs-on: ubuntu-16.04
+    steps:
+    - name: Checkout sources
+      uses: actions/checkout@v2
+    - name: Setup Python
+      uses: actions/setup-python@v1
+      with:
+        python-version: 3.5
+    - name: Install dependencies
+      run: |
+        sudo apt update && sudo apt install -y enchant
+        pip install -U tox setuptools
+    - name: Run tests
+      run: tox -e unittest
+
+  backwards_compatability_test:
+    runs-on: ubuntu-16.04
+    env:
+      TOXENV: py35-pypi
+      ROS_DISTRO: kinetic
+      TEST: backwards
+    steps:
+    - name: Checkout sources
+      uses: actions/checkout@v2
+    - name: Setup Python
+      uses: actions/setup-python@v1
+      with:
+        python-version: 3.5
+        architecture: x64
+    - name: Install dependencies
+      run: |
+        sudo apt update
+        sudo apt install -y enchant
+    - name: Install Tox
+      run: pip install tox
+    - name: Run tests
+      run: tox
```

Comment:
Could you either use `TOXENV` or `tox -e <env>` consistently on all the tests? Just for readability.

Reply:
offhand it seems like `TOXENV` in the top level `env:` block may be more readable because it's right up top near the name of the test